cluster-up/tags

```diff
@@ -0,0 +1,34 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
can you explain what exactly this file is doing?

Reply:
ha, this is my ctags file used by my editor (vim).  It wasn't supposed to be committed.